.github/workflows/test-chart-version-nightly-template.yaml

```diff
@@ -0,0 +1,70 @@
+# Basic test for maintenance Camunda 8 Helm charts.
+name: "Test - Chart Version Nightly - Template"
+
+on:
+  workflow_call:
+    inputs:
+      chart-version:
+        description: Chart version to test
+        required: true
+        type: string
+      scenarios:
+        description: Scenarios
+        required: true
+        type: string
+      type:
+        description: Type
+        required: true
+        type: string
+
+concurrency:
+  group: ${{ github.workflow }}-${{ inputs.chart-version }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
+  validation:
+    name: Camunda ${{ inputs.chart-version }} - Validation
+    uses: ./.github/workflows/chart-validate-template.yaml
+    with:
+      identifier: "${{ inputs.type }}-vald-${{ inputs.chart-version }}"
+      camunda-helm-dir: "camunda-platform-${{ inputs.chart-version }}"
+      camunda-helm-git-ref: "${{ github.event.pull_request.head.sha }}"
+  format-scenarios:
+    needs: format-scenarios
+    runs-on: ubuntu-latest
+    env:
+      input: ${{ inputs.scenarios }}
+    outputs: 
+      scenarios-formatted: ${{ steps.make-json-array.outputs.json_array }}
+    steps:
+      - name: Convert string to JSON array and set as output
+        id: make-json-array
+        run: |
+          # For example, this converts an input "multitenancy, oidc" to ["multitenancy","oidc"]
+          json=$(printf '%s' "$input" | awk -F',' '{for(i=1;i<=NF;i++){gsub(/^ +| +$/,"",$i); printf "%s\"%s\"", (i==1?"[":",") , $i} printf "]\n"}')
```

Comment:
This can be done better/cleaner with JQ.

Reply:
maybe something like this:
```
echo "$input" | jq -R -c 'split(",")'
```

Reply:
What I mentioned is working. I replace the command.